config/locales/alchemy.it.yml

```diff
@@ -0,0 +1,924 @@
+
+    essence_pictures:
+      css_classes:
+        left: 'Alla sinistra del testo'
+        right: 'Alla destra del testo'
+        no_float: 'Sopra al testo'
+
+    # == Contactform translations
+    contactform:
+      labels:
+        salutation: 'Salve'
+        choose: 'Prego scelga'
+        mr: 'Sig.'
+        mrs: 'Sig.ra'
+        firstname: 'Nome'
+        lastname: 'Cognome'
+        address: 'Via / N.'
+        zip: 'CAP / Città'
+        phone: 'Telefono'
+        email: 'Email'
+        message: 'Messaggio'
+        send: 'Invia'
+        mandatory_fields: '*Campi obbligatori.'
+      # The flash message shown after successfully sending the message.
+      messages:
+        success: 'Il suo messaggio è stato consegnato con successo.'
+
+    # == Translated language names for translation select
+    translations:
+      de: 'DE'
+      en: 'EN'
+      nl: 'NL'
+      fr: 'FR'
+      it: 'IT'
+
+    # == User roles translations
+    user_roles:
+      registered: "Registrato"
+      member: "Membro"
+      author: "Autore"
+      editor: "Editore"
+      admin: "Amministratore"
+
+    # == Mime Types translations
+    # These are human readable mime types used for the document-type row in archive files.
+    mime_types:
+      audio/mp4: 'Audio-MP4'
+      application/msword: 'Documento-Word'
+      application/rtf: 'Documento-RTF'
+      audio/mpeg: 'Audio-MP3'
+      text/plain: 'Documento-di-Testo'
+      video/mp4: 'Video-MP4'
+      video/mpeg: 'Video-MPEG'
+      application/pdf: 'Documento-PDF'
+      application/x-flash-video: 'Video-Flash'
+      video/x-flv: 'Video-Flash'
+      application/x-shockwave-flash: 'Video-Flash'
+      application/zip: 'Archivio-ZIP'
+      application/x-rar: 'Archivio-RAR'
+      application/vnd:
+        ms-excel: 'Documento-Excel'
+      video/quicktime: 'Video-Quicktime'
+      image/x-psd: 'File-Photoshop'
+      image/gif: 'Immagine-GIF'
+      image/png: 'Immagine-PNG'
+      image/jpeg: 'Immagine-JPG'
+      video/x-msvideo: 'Video-AVI'
+      video/x-ms-wmv: 'Windows Media Video'
+      image/tiff: 'Immagine-TIFF'
+      'text/x-vcard': 'vCard'
+      application/vcard: 'vCard'
+
+    link_target_options:
+      default: Stessa Finestra
+      blank: Nuova Finestra/Tab
+      overlay: Sovrapposta
+
+    resource_help_texts:
+      site:
+        aliases: Prego separa i domini aggiuntivi con uno spazio od un ritorno a capo.
+
+    anchor: 'Collegamento'
+    back: 'indietro'
+    create_tree_as_new_language: "Crea un nuovo albero della lingua per %{language}"
```

Comment:
@kennyadsl it sounded strange to me as well.

I'll put in some alternatives:

* ramificazione
* alberatura
* struttura